docs/add-runtime.md

```diff
@@ -0,0 +1,250 @@
+Next create a shell script yourself with your language name. As an example, the shell script name for dart would be `package-dart.sh`
+
+Within this newly created script copy-paste this script and replace all the `LANGUAGE_NAME` parts with your language's name
+```
+echo  'LANGUAGE_NAME Packaging...'
+rm $(pwd)/tests/resources/LANGUAGE_NAME.tar.gz
+tar -zcvf $(pwd)/tests/resources/LANGUAGE_NAME.tar.gz -C $(pwd)/tests/resources/LANGUAGE_NAME .
+```
+Then save this file. Then `cd` into the root of the `php-runtimes` project in a terminal. Run the following command replacing the `LANGUAGE_NAME` with your language's name:
+```
+chmod +x ./tests/resources/package-LANGUAGE_NAME.sh && ./tests/resources/package-LANGUAGE_NAME.sh
+```
+This command adds execution permissions to your script and executes it.
+
+NOTE: If you ever want to repackage your script you can simply run: `./tests/resources/package-LANGUAGE_NAME.sh` in the root of the `php-runtimes` project since you don't have to change permissions more than once.
+
+### 5.3 Adding your runtime to the main testing script
+Now you have created your test execution script and have packaged it up for your runtime to execute you can now add it to the main testing script. Open up the `./tests/Runtimes/RuntimesTest.php` file and find the part where we are defining `$this->tests`.
+
+Once you have found this, Add your own entry into this array like so:
+```php
+'LANGUAGE_NAME-VERSION'  =>  [
+    'code'  =>  $functionsDir .  ' /LANGUAGE_NAME.tar.gz',
+    'entrypoint'  =>  'Test file', // Replace with the name of the test file you wrote in ./tests/resources/LANGUAGE_NAME
+    'timeout'  =>  15,
+    'runtime'  =>  'LANGUAGE_NAME-VERSION',
+    'tarname'  =>  'LANGUAGE_NAME-VERSION.tar.gz', // Note: If your version has a point in it replace it with a dash instead for this value.
+],
+```
+Make sure to replace all instances of `LANGUAGE_NAME` with your language's name and `VERSION` with your runtime's version.
+
+Once you have done this and saved it, it is finally time to move onto one of the final steps.
+
+### 5.4 Running the tests.
+Running the tests is easy, simply run `docker-compose up` in the root of the `php-runtimes` folder. This will launch a Docker container with the test script and start running through all the runtimes making sure to test them thoroughly.
+
+If all tests pass then congratulations! You can now go ahead and file a PR against the `php-runtimes` repo making sure to target the `refactor` branch, make sure you're ready to respond to any feedback which can arise during our code review.
+
+## 6. Raise a pull request
+First of all, commit the changes with the message `Added XXX Runtime` and push it. This will publish a new branch to your forked version of OpenRuntimes. If you visit it at `github.com/YOUR_USERNAME/php-runtimes`, you will see a new alert saying you are ready to submit a pull request. Follow the steps GitHub provides, and at the end, you will have your pull request submitted.
```

Comment:
```suggestion
First of all, commit the changes with the message `Added XXX Runtime` and push it. This will publish a new branch to your forked version of Open Runtimes. If you visit it at `github.com/YOUR_USERNAME/php-runtimes`, you will see a new alert saying you are ready to submit a pull request. Follow the steps GitHub provides, and at the end, you will have your pull request submitted.
```